src/Merchants/Content/Merchant/Api/ProfileController.php

```diff
@@ -26,15 +28,25 @@ class ProfileController
      * @var EntityRepositoryInterface
      */
     private $merchantRepository;
+
     /**
      * @var DataValidator
      */
     private $dataValidator;
 
-    public function __construct(EntityRepositoryInterface $merchantReepository, DataValidator $dataValidator)
-    {
+    /**
+     * @var StorefrontMediaUploader
+     */
+    private $uploader;
+
+    public function __construct(
+        EntityRepositoryInterface $merchantReepository,
```

Comment:
`$merchantRepository`

Reply:
If it's just that, I will change it in the next PR, ok?

Reply:
On the other hand, I just added delete here too :)